.github/workflows/tests-weekly.yaml

```
@@ -17,6 +17,42 @@ on:
         default: false
 
 jobs:
+  lxd:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-22.04, ubuntu-22.04-arm, ubuntu-24.04, ubuntu-24.04-arm]
```

```suggestion
        os: [ubuntu-22.04, ubuntu-24.04]
```

Arm runners not working?